### PR TITLE
(feat) Add support for local configuration in development environment

### PR DESCRIPTION
**Pull request by chibongho**
# Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. Ensure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing#contributing-guidelines) label (such as `feat`, `fix`, or `chore`, among others). See existing PR titles for inspiration.

## For changes to apps

- [ ] My work conforms to the [**O3 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://om.rs/o3ui).

## If applicable

- [ ] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

### CODE REVIEW

**Review by brandones — APPROVED** [review]
Looks good, thanks @chibongho !

File: packages/tooling/openmrs/src/commands/develop.ts [hunk 1]
```diff
@@ -116,6 +132,13 @@
   // Route for custom `index.html` goes above static assets
   app.get(indexHtmlPathMatcher, (_, res) => res.contentType('text/html').send(indexContent));
 
+  configPaths.forEach((path, i) => {
+    const url = localConfigUrls[i];
+    app.get(url, (_, res) => {
+      res.contentType('application/json').send(readFileSync(resolve(path)));
+    });
```
[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/tooling/openmrs/src/commands/develop.ts:139]: Something to be worried about?

[the remark this answers is not in the document]
Comment (chibongho) [reply packages/tooling/openmrs/src/commands/develop.ts:139]: I doubt it given that it should only ever be used in a local dev environment.

**Comment by chibongho** [comment]
> Looks good, thanks @chibongho !

Thanks @brandones . I made this a few months ago, but can no longer test it as `npx openmrs develop` no longer works on my machine for some reason. I made this draft PR to have @ibacher try to get it over the finish line.

If I'll keep this unmerged for now until someone verifies that it still works fine.

File: packages/tooling/openmrs/src/commands/develop.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ibacher) [reply packages/tooling/openmrs/src/commands/develop.ts:139]: Yeah, there were a few of these for similar code.

**Comment by ibacher** [comment]
I made some other adjustments (renaming path -> file), but AFAICT, the only thing we really needed here was this:

```diff
-     res.contentType('application/json').send(readFileSync(resolve(path)));
+     res.contentType('application/json').send(readFileSync(resolve(process.cwd(), file)));
```

**Comment by ibacher** [comment]
PS I think from core `yarn openmrs develop` works better than `npx`

**Pull request merged**